src/compiler/scala/tools/nsc/typechecker/Typers.scala

```diff
@@ -959,6 +959,7 @@ trait Typers extends Adaptations with Tags with TypersTracking with PatternTyper
           doIt
         }
 
+        def isAnyDerived(sym: Symbol): Boolean = AnyTpe.nonPrivateMember(sym.name) != NoSymbol
```

Comment:
TIL `isUniversalMember(sym)`, which is more precise.

So `sym.overrides.exists(isUniversalMember)`.

Or `sym.overrides.lastOption.map(isUniversalMember).getOrElse(false)`.

`class C { def equals() = false } ; new C().equals` for example. Hopefully their lint for overloading `equals` also alerted.

Reply:
Members get the secret fifth [churro flavor](https://www.universalstudioshollywood.com/park-tips/4-all-new-churro-flavors) at our theme park.

Reply:
Addressed in d090b8002bb965f1e6f11f82bcf26417aead5636. `meth.overrides` returned `Nil` for value class's `hashCode` so I went with `!isUniversalMember(meth.overrides.lastOption.getOrElse(meth))`.